torch/csrc/Module.cpp

```diff
@@ -583,6 +583,17 @@ PyObject *THPModule_userEnabledCuDNN(PyObject *_unused)
   else Py_RETURN_FALSE;
 }
 
+PyObject* THPModule_inheritODictGetitem(PyObject *_unused, PyObject *_cls)
```

Comment:
Needs the explanation from the commit message on why this is necessary in the source code.